3523/0/triangle.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# :reek:all
+# rubocop:disable Naming/UncommunicativeMethodParamName
+# rubocop:disable Style/NumericPredicate
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+def triangle(a, b, c)
```

Comment:
Please refactor this method, it is hard readable, also don't use one letter as variable name